ssg/build_yaml.py

```diff
@@ -1044,6 +1044,7 @@ class Rule(object):
     }
 
     PRODUCT_REFERENCES = ("stigid", "cis",)
+    GLOBAL_REFERENCES = ("srg", "disa", "cis-csc",)
```

Comment:
Should `vmmsrg` be added here as well since you fixed it in the `set_password_hashing...` rule?

Reply:
probably yes ;)

Reply:
Done

Reply:
I think we can add more in the future as we understand more about the references